sycl/include/sycl/builtins.hpp

```diff
@@ -1433,8 +1433,7 @@ any(T x) __NOEXC {
 
 // int any (vigeninteger x)
 template <typename T>
-detail::enable_if_t<detail::is_vigeninteger<T>::value, detail::anyall_ret_t>
```

Comment:
Do we still have uses of `detail::anyall_ret_t` after this change?

Reply:
That's right.